packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -187,6 +190,7 @@ class FlutterDevice {
     CompileExpression compileExpression,
     ReloadMethod reloadMethod,
     GetSkSLMethod getSkSLMethod,
+    PrintErrorEventMethod printErrorEventMethod,
```

Comment:
This needs to be hooked up somewhere to work, are either of the Hot/Cold runners passing this through?

Reply:
They both call `connectToServiceProtocol` defined on `ResidentRunner` in their `attach` methods, so that should mean they use the `printErrorEventMethod` defined on `ResidentRunner` too. Are there other places that need to pass in a print method too?